Makefile

```diff
@@ -0,0 +1,776 @@
+# Makefile.in generated by automake 1.16.5 from Makefile.am.
```

Comment:
Makefile is a generated file. I think we should not check in the generated files.

Reply:
Removed